Sprint-1/implement/max.js

```diff
@@ -1,4 +1,14 @@
 function findMax(elements) {
+  const filterNumberArray = [];
+  for (let i = 0; i < elements.length; i++) {
+    if (typeof elements[i] == "number") {
+      filterNumberArray.push(elements[i]);
+    }
+  }
+  const maxValue = Math.max(...filterNumberArray);
+  return maxValue;
 }
+// console.log(findMax([1, 2, 3, 4, "hello"]));
+// console.log(findMax([]));
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax([NaN])
findMax([0, NaN, 1])
```